### PR TITLE
Dead usb peak

### PR DESCRIPTION
**Pull request by DeadUSB**
DO IT.

### CODE REVIEW

**Comment by DeadUSB** [comment]
do it

**Comment by devkarthin** [comment]
Approved👍

@JoeMama-SK Please merge ASAP

File: README.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ Karthin Srinavakili
 
 
 
-forked by deadusb (cool)
+forked by deadusb (coolest)
```
Comment (devkarthin) [inline README.md:8]: SO TRUE!

**Review by devkarthin — APPROVED** [review]
I like it😀👍

**Review by devkarthin — CHANGES_REQUESTED** [review]
Hi

**Comment by sppongy** [comment]
# @`1. ### [_****_]([url](url))`

**Comment by devkarthin** [comment]
> # @`1. ### [_****_]([url](url))`

🤯🤯🤯

**Pull request merged**